extras/create_snapshots_reports_scorecard.py

```diff
@@ -326,28 +337,26 @@ def generate_weekly_snapshots(db, cyhy_db_section):
     reports_to_generate = create_list_of_reports_to_generate(db)
 
     logging.info("Building list of snapshots to generate...")
+    global snapshots_to_generate
     snapshots_to_generate = create_list_of_snapshots_to_generate(
         db, reports_to_generate
     )
 
+    logging.debug("%d snapshots to generate: %s", len(snapshots_to_generate), snapshots_to_generate)
+
```

Comment:
I think it's worth putting a comment here stating that we don't need to do any thread locking against `snapshots_to_generate` because the code is single-threaded at this point.  Otherwise this point could needlessly confuse us in the future.

Reply:
Good thinking - please see https://github.com/cisagov/cyhy-reports/pull/106/commits/c2a99d6b63fbba6c4b4a6072600c02e2c6b85fd2.